packages/xo-web/src/common/xo/migrate-vm-modal/index.js

```diff
@@ -86,6 +136,17 @@ export default class MigrateVmModalBody extends BaseComponent {
           : false
     )
 
+    this._getSnapashotSrPredicate = createSelector(
+      () => this.state.host,
+      host =>
+        host
+          ? snapshotsSr =>
+              isSrWritable(snapshotsSr) &&
+              (snapshotsSr.$container === host.id ||
+                snapshotsSr.$container === host.$pool)
```

Comment:
If it's the exact same predicate as the VMs' one, you don't need another selector.